form.php

```diff
@@ -482,9 +482,10 @@ public function onFormProcessed(Event $event)
                 break;
             case 'redirect':
                 $this->grav['session']->setFlashObject('form', $form);
-                $url = ((string)$params);
+                $url = (string)(is_array($params) ? $params['route'] : $params) ?? $this->grav['page']->url();
```

Comment:
`$url = (string)(is_array($params) ? ($params['route'] ?? null) : $params) ?: $this->grav['page']->url();`